src/pages/Pathway/Pathway.js

```diff
@@ -8,12 +8,16 @@ import resources from '../../data/resources';
 import './Pathway.css';
 
 const Pathway = () => {
+    // inside create pathway function have something like :
+    // setPathway(example_resources ? example_resources : pathway); // if a new one is generated, take it; otherwise take the last one generated
     const {quizResults} = useContext(QuizContext);
     // start from here...
     console.log(quizResults);
-    
-    const example_resources = resources[3].links;
 
+    let pathway = localStorage.getItem('pathwayData') ? JSON.parse(localStorage.getItem('pathwayData')) : resources[2].links; 
```

Comment:
I wonder if we could reorganize this part to be something like:
```
if (!localStorage.getItem('pathwayData') // remove this if statement and keep else statement, once there is a function to ensure that there is a pathway
    let pathway = resources[2].links;
    localStorage.setItem('pathwayData', JSON.stringify(pathway));
} else let pathway = JSON.parse(localStorage.getItem('pathwayData'));
```
lmk what you think!